.travis.yml

```diff
@@ -1,40 +1,125 @@
+conditions: v1
+
 language: python
+python:
```

Comment:
3.7?

Reply:
It's included separately because of Travis specifics. See build page, it's there.